packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.scss

```diff
@@ -125,4 +125,82 @@
   min-width: 180px;
 }
 
+.neutralDark {
```

Comment:
This components should be glamorized. Also, it's unclear to me why the creator of a SwatchColorPicker wouldn't just use theme.palette.<name> (which returns a string) when building the component instead of having to redefine the themeColors

Reply:
The problem with that is how would you know the color belongs to theme, that would just apply the color and won't be themed. Which is something that we don't want. Or is there a way of translating it to theme color that I am not aware of?

Reply:
The component shouldn't know about themes or a specific set of colors. The colors are passed in as props to the component. Those colors could happen to come from the theme, but the component doesn't need to do that. Your app would just need to make sure that the colors passed into the swatch picker are updated when the theme changes.

Reply:
I have updated the code. Please take a look